python/ray/data/random_access_dataset.py

```diff
@@ -56,8 +57,13 @@ def __init__(
                 self._upper_bounds.append(b[1])
 
         logger.info("[setup] Creating {} random access workers.".format(num_workers))
+        ctx = DatasetContext.get_current()
+        if ctx.scheduling_strategy is not None:
```

Comment:
Should this be `ctx.scheduling_strategy != DEFAULT_SCHEDULING_STRATEGY`

Reply:
Ah yep this is a holdover from the old default, I'll change this!